src/frontends/onnx/frontend/src/op/com.microsoft/fused_conv.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/com.microsoft/fused_conv.hpp"
+
+#include <memory>
+#include <vector>
+
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+#include "op/conv.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector fused_conv(const Node& node) {
+    auto conv_res = conv(node).at(0);
+
+    if (node.get_ng_inputs().size() == 4) {  // Z input provided
```

Comment:
It's worth to add `NullNode` check
`if (node.get_ng_inputs().size() == 4 && !op::is_null(inputs[3]))`